cmd/cluster-agent/app/app.go

```diff
@@ -194,6 +198,27 @@ func start(cmd *cobra.Command, args []string) error {
 		log.Debugf("Health check listening on port %d", healthPort)
 	}
 
+	// Attempt to create and start the core remote config service, creating the local client only if that succeeds
+	var configService *remoteconfig.Service
+	var rcClient *remote.Client
+	if config.Datadog.GetBool("remote_configuration.enabled") {
+		configService, err = remoteconfig.NewService()
+		if err == nil {
+			if err := configService.Start(context.Background()); err == nil {
+				rcClient, err = remote.NewClient("cluster-agent", configService, version.AgentVersion, []data.Product{data.ProductTesting1}, time.Second*5)
+				if err != nil {
+					log.Errorf("Failed to start local config management service client: %s", err)
+				} else {
+					rcClient.Start()
+				}
+			} else {
+				log.Errorf("Failed to start config management service: %s", err)
+			}
+		} else {
+			log.Errorf("Failed to initialize config management service: %s", err)
+		}
```

Comment:
Can we [align the happy path to the left](https://medium.com/@matryer/line-of-sight-in-code-186dd7cdea88)?
(i.e `if err != nil` instead of if `err == nil`)